tests/test_binaries.py

```diff
@@ -146,17 +146,18 @@ def test_logger_verbosity(self):
 
         dirname = os.path.dirname(__file__)
         filename = os.path.join(dirname, 'binaries', 'basic_linux_amd64')
-        output = subprocess.check_output([PYTHON_BIN, '-m', 'manticore', filename])
+        output = subprocess.check_output([PYTHON_BIN, '-m', 'manticore', '--no-colors', filename])
         output_lines = output.splitlines()
-        start_info = output_lines[:2]
-        testcase_info = output_lines[2:-5]
-        stop_info = output_lines[-5:]
 
-        self.assertEqual(len(start_info), 2)
-        self.assertEqual(len(stop_info), 5)
+        self.assertEqual(len(output_lines), 6)
+
+        self.assertIn(b'Loading program', output_lines[0])
+        self.assertIn(b'Verbosity set to 1.', output_lines[1])
+        self.assertIn(b'Generated testcase No. 0 - Program finished with exit status: 0', output_lines[2])
+        self.assertIn(b'Generated testcase No. 1 - Program finished with exit status: 0', output_lines[3])
+        self.assertIn(b'Results in ', output_lines[4])
+        self.assertIn(b'Total time: ', output_lines[5])
 
-        for line in testcase_info:
```

Comment:
Uh, this for here was wrong, the `testcase_info` was empty because there were not that much output to account for `[2:-5]` slice. As a result, the assertion below never fired.

But the test was helpful anyway.